pages/glossary/programmatic-label.md

```diff
@@ -3,6 +3,8 @@ title: Programmatic Label
 key: programmatic-label
 unambiguous: true
 objective: true
+input_aspects:
+  - DOM tree
```

Comment:
Labelled controls can not be hidden with CSS.

```suggestion
  - CSS styling
  - DOM tree
```

Reply:
🤔 The  visibility of the labelled control is currently handled in the rule's applicability (definition is only used in one rule), and the HTML definition of "labelled control" does not speak about visibility.

We could move the visibility to the definition (and require that the programmatic label is visible), but I don't think that it is necessarily. The current definition can be used for "AT specific" labels, which could be useful in other rules.